src/api/controllers/guilds.ts

```diff
@@ -52,6 +52,21 @@ export async function handleInternalGuildDelete(data: DiscordPayload) {
     }
   });
 
+  cacheHandlers.forEach("members", async (member) => {
+    if (!member.guilds.has(payload.id)) {
+      await cacheHandlers.delete("members", member.id);
+    }
```

Comment:
I'm so sorry, I selected the wrong line on my review.

```suggestion
    if (!member.guilds.has(payload.id)) return;
```